### PR TITLE
Fix/recharts

### PR DESCRIPTION
**Pull request by emersonlaurentino**
#### What is the purpose of this pull request?
fix: https://github.com/vtex/styleguide/issues/1043

#### What problem is this solving?
fix: https://github.com/vtex/styleguide/issues/1043

#### How should this be manually tested?
check size in bundlephobia

[@vtex/styleguide@9.111.0](https://bundlephobia.com/result?p=@vtex/styleguide@9.111.0)

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "vendor": "vtex",
   "name": "styleguide",
   "title": "VTEX Styleguide",
-  "version": "9.111.4",
+  "version": "9.111.5-beta",
```
Comment (arthurepc) [inline manifest.json]: Do you have any specific reason for this to be a beta tag?

Comment (emersonlaurentino) [reply manifest.json]: test size in bundlephobia

https://bundlephobia.com/result?p=@vtex/styleguide@9.111.5-beta

Comment (emersonlaurentino) [reply manifest.json]: but `releasy` has a problem, he publish beta as latest!

Comment (emersonlaurentino) [reply manifest.json]: Screen Shot 2020-02-17 at 19 28 52

**Comment by emersonlaurentino** [comment]
We have a problem, if we put `recharts` as peerDependencies solves our problem with the size of the lib, however our documentation will break because we will not be able to find the dependency, consequently we would need to add `recharts` to devDependencies, which would make the lib big again.

File: react/package.json [hunk 2]
```diff
@@ -26,6 +24,12 @@
     "@types/node": "^12.6.2",
     "@types/react": "^16.8.23",
     "@types/react-dom": "^16.9.0",
-    "@types/recharts": "^1.8.0"
+    "@types/recharts": "^1.8.0",
+    "recharts": "^2.0.0-beta.1"
```
Comment (arthurepc) [inline react/package.json]: Also, do you have some specific reason to use this beta version of recharts?

Comment (arthurepc) [reply react/package.json]: And should this be in both `devDependencies` and `peerDependencies`?

Comment (emersonlaurentino) [reply react/package.json]: > Also, do you have some specific reason to use this beta version of recharts?

The beta version was already being used, the Fanny used it, so I didn't want to change the version, just put it as peerDependencies.

> And should this be in both devDependencies and peerDependencies?

I explained in my last comment https://github.com/vtex/styleguide/pull/1048#issuecomment-587179160

Comment (emersonlaurentino) [reply react/package.json]: the ideal scenario would be for our documentation to be separate from our styleguide.

**Pull request merged**